src/controllers/event.js

```diff
@@ -0,0 +1,16 @@
+const createModal = require ( '../models/events.js');
+const {db} = require ('../databaseSetup');
+const {DataTypes} = require ('sequelize');
+
+const Events = createModal(db, DataTypes);
+
+const getAllEvents = async (req, res) => {
+    try {
+        const events = await Events.findAll();
+        res.json(events);
+    } catch (error) {
+        res.json({ message: error.message });
+    }  
+}
+
+module.exports = getAllEvents;
```

Comment:
Suggest to not doing default export. Right now it prevent us add more handlers in this file